var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import os
+from spack import *
+
+
+def _verbs_dir():
+    """Try to find the directory where the OpenFabrics verbs package is
+    installed. Return None if not found.
+    """
+    try:
+        # Try to locate Verbs by looking for a utility in the path
+        ibv_devices = which("ibv_devices")
+        # Run it (silently) to ensure it works
+        ibv_devices(output=str, error=str)
+        # Get path to executable
+        path = ibv_devices.exe[0]
+        # Remove executable name and "bin" directory
+        path = os.path.dirname(path)
+        path = os.path.dirname(path)
+        # There's usually no "/include" on Unix; use "/usr/include" instead
+        if path == "/":
+            path = "/usr"
+        return path
+    except TypeError:
+        return None
+    except ProcessError:
+        return None
+
+
+class Gpi2(AutotoolsPackage):
+    """GPI-2 is an API for the development of scalable, asynchronous and fault
+    tolerant parallel applications. It implements the GASPI specification
+    (www.gaspi.de)"""
+
+    homepage = "http://www.gpi-site.com"
+    url      = "https://github.com/cc-hpc-itwm/GPI-2/archive/v1.4.0.tar.gz"
+    git      = "https://github.com/cc-hpc-itwm/GPI-2.git"
+
+    version('master', branch='master')
+    version('next', branch='next')
```

Comment:
This version will sort as older than all stable releases, we only make exceptions for `develop`, `master`, `head`, and `trunk`.

Reply:
I'm not sure that I understood this one. You mean that next branch shouldn't be specified here or?

Reply:
The branch must be specified, but you can name the version anything you want. I would recommend:
```suggestion
    version('master', branch='next')
```
if `next` is newer than stable releases.

Reply:
I think this is a good idea. I don't know what was developers intention but there are no commits to the master for quite some time.

Reply:
You can also do:
```python
version('develop', branch='next')
version('master', branch='master')
```
It really depends on which branches you plan on using, you don't have to add any at all if you don't care.

Reply:
I don't know what is the plan of developers with the master branch and therefore I will return it and add develop version from your last suggestion.